cmd/kops/create_cluster.go

```diff
@@ -59,6 +59,8 @@ type CreateClusterOptions struct {
 	MasterSize           string
 	MasterCount          int32
 	NodeCount            int32
+	MasterVolumeSize     int32
```

Comment:
@WillemMali I think you might be confused on what this option actually does. It will set the root volume size on the underlying master or worker instances.  These volumes are not 'on' the master -- they are the default master volumes, which everything else will live on. These volumes could obviously never be 100MB or even in the MB range as a default install of the OS is much more than that. The default size given by AWS is 9GB since thats about the minimum an instance could run on.

@chrislovecnm I could work on the other defaults/updating tests as well when i have some time. This will be a separate PR as stated, and won't affect this PR.